marconi/src/Marconi/IndexersHotStore.hs

```diff
@@ -0,0 +1,172 @@
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE GADTs                #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+{- | IndexerHotstore provides in concurrent in memory cache for marcoin clients
```

Comment:
marcoin -> Marconi

Reply:
provides in concurrent?

Reply:
typo, should've been  "provides a concurrent", will fix